server/processors/src/main/java/org/hawkular/apm/server/processor/nodedetails/NodeDetailsDeriver.java

```diff
@@ -116,7 +123,8 @@ protected void deriveNodeDetails(Trace trace, List<Node> nodes, List<NodeDetails
                     nd.setHostName(trace.getHostName());
                 }
 
-                nd.setProperties(trace.allProperties());
```

Comment:
State before: all properties from all nodes was added to the NodeDetails. However there is a bug in `Trace#allProperties()`  [1]. It adds properties only from direct nodes (we currently use only the first one)

Now: only common and actual node's properties are added to NodeDetails'


[1]: https://github.com/hawkular/hawkular-apm/blob/master/api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java#L200